spec/features/secure_routes_spec.rb

```diff
@@ -49,13 +49,13 @@
     it "should not provide access to regular users" do
       login_as user, scope: :user
       visit '/about/health'
-      expect(page).to have_content('Sample Content')
+      expect(page).to have_content('Featured Content')
     end
 
     it "should not provide access to anonymous users" do
       login_as user, scope: :user
       visit '/about/health'
-      expect(page).to have_content('Sample Content')
+      expect(page).to have_content('Featured Content')
```

Comment:
Local change to the splash page, missed this in the upstream code merge. Not certain why the tests for secure routes depend on expectations that the resulting page has a certain content string.
There's a better way in Ruby, right? I'll search Jupiter when I have time.

Reply:
I can think a couple of better ways to do that, yeah, depending on what the implementation is like. Usually what you'd do is redirect the unauthorized user to a message or login page or whatever when they visit the secure URL, and then in the test just assert that you got redirected to where you expected them to be sent.